pdsClient/pdsClient_test.go

```diff
@@ -824,6 +824,8 @@ func TestFindModifiedRecords(t *testing.T) {
 			Before: endTime,
 		},
 	}
+
+	// TODO: add retry logic, new ES recoreds not available immediatly
```

Comment:
This test is still failing :(
I need to add retry logic so it can wait for the data to commit to ES but plan to do that after getting historical reconciler finished

Reply:
`recoreds` -> `records`?